torch/quantization/_numeric_suite.py

```diff
@@ -218,21 +244,128 @@ def prepare_model_with_stubs(float_module, q_module, module_swap_list, Logger):
         q_module._modules[key] = value
 
 
-def compare_model_stub(float_model, q_model, module_swap_list, data, Logger=Logger):
-    r"""Returns a dict with key corresponding to module names and each entry being
-    a dictionary with two keys 'float' and 'quantized', containing the output
-    tensors of quantized and its matching float shadow module. This dict can be
-    used to compare and compute the module level quantization error.
+def compare_model_stub(
+    float_model, q_model, module_swap_list, data, Logger=ShadowLogger
+):
+    r"""Compare quantized module in a model with its floating point counterpart,
+    feeding both of them the same input. Return a dict with key corresponding to
+    module names and each entry being a dictionary with two keys 'float' and
+    'quantized', containing the output tensors of quantized and its matching
+    float shadow module. This dict can be used to compare and compute the module
+    level quantization error.
+
+    This function first call prepare_model_with_stubs() to swap the quantized
+    module that we want to compare with the Shadow module, which takes quantized
+    module, corresponding float module and logger as input, and creates a forward
+    path inside to make the float module to shadow quantized module sharing the
+    same input. The logger can be customizable, default logger is ShadowLogger
+    and it will save the outputs of the quantized module and float module that
+    can be used to compute the module level quantization error.
+
+    Example usage:
+        module_swap_list = [torchvision.models.quantization.resnet.QuantizableBasicBlock]
+        ob_dict = compare_model_stub(float_model,qmodel,module_swap_list, data)
+        for key in ob_dict:
+            print(key, compute_error(ob_dict[key]['float'], ob_dict[key]['quantized'].dequantize()))
 
     Args:
-        float_module: the float module used to generate the q_module
-        q_module: the quantized module
-        module_swap_list: list of float module types to attach the shadow
-        Logger: the class to be used in shadow module to process the outputs of
+        float_model: float model used to generate the q_model
+        q_model: model quantized from float_model
+        data: input data used to run the prepared q_model
+        module_swap_list: list of float module types at which shadow modules will
+            be attached.
+        Logger: type of logger to be used in shadow module to process the outputs of
             quantized module and its float shadow module
     """
     prepare_model_with_stubs(float_model, q_model, module_swap_list, Logger)
     q_model(data)
     ob_dict = {}
     get_observer_dict(q_model, ob_dict, Logger)
     return ob_dict
+
```

Comment:
NIT: Do we need to create an empty dict and pass it to get_observer_dict? Can we instead do:
ob_dict = get_observer_dict(q_model, Logger)?